CRM/Friend/BAO/Friend.php

```diff
@@ -143,27 +142,26 @@ public static function create(&$params) {
 
       //create contact only if it does not exits in db
```

Comment:
could fix the comment typo while you're changing this   'does not exits' => 'does not exist'